front-end/src/temperature/edit_temperature.jsx

```diff
@@ -431,26 +431,6 @@ const EditTemperature = ({
             </div>
           </div>
         </div>
-        <div className='row'>
```

Comment:
@ranjib  @prsws was removing the hysteresis input here intended?

Reply:
No